examples/snake/main.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2020 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build example jsgo
+
+package main
+
+import (
+	"fmt"
+	"image/color"
+	"log"
+	"math/rand"
+
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/ebitenutil"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+	gridSize     = 10
+	xNumInScreen = screenWidth / gridSize
+	yNumInScreen = screenHeight / gridSize
+)
+
+const (
+	dirNone = iota
+	dirLeft
+	dirRight
+	dirDown
+	dirUp
+)
+
```

Comment:
Set the random seed, or you will see always the same result:

```
func init() {
    rand.Seed(time.Now().UnixNano())
}
```

Reply:
Thanks